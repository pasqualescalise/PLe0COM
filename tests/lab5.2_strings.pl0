VAR str[100] : char;

BEGIN
	str := "Hello World!";
	print str;
	str := "";
	print str;
	str := "\"H\"e\"l\"l\"o\"\" \"\"W\"o\"r\"l\"d\"!\"";
	print str;
	str := "
		Hello
		World
		!
	";
	print str;
	print "Hello inlined World!";
	print "This has a /* comment */ inside // actually two"
END
