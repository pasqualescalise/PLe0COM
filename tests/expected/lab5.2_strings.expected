Hello World!

"H"e"l"l"o"" ""W"o"r"l"d"!"

		Hello
		World
		!
	
Hello inlined World!
This has a /* comment */ inside // actually two
